tad/api/routes/pages.py

```diff
@@ -0,0 +1,25 @@
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, Request
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+from tad.services.statuses import StatusesService
+from tad.services.tasks import TasksService
+
+router = APIRouter()
+templates = Jinja2Templates(directory="tad/site/templates")
+
+
+@router.get("/", response_class=HTMLResponse)
```

Comment:
something to consider: 
* make the get dynamic with a resource_id and use that in the name option of TemplateReponse

Reply:
Sounds good for the next iteration, I am not sure how to do that.